birdsong/conf.py

```diff
@@ -1,3 +1,32 @@
 from django.conf import settings
+import django.utils.module_loading as module_loading
+from django.utils.translation import gettext as _
+
+### BIRDSONG_CONTACT ###
+BIRDSONG_CONTACT_MODEL = getattr(settings, 'BIRDSONG_CONTACT_MODEL', 'birdsong.Contact')
+
+### BIRDSONG_TEST_CONTACT ###
+BIRDSONG_TEST_CONTACT = getattr(settings, 'BIRDSONG_TEST_CONTACT', { 'email': 'wagtail.birdsong@example.com' })
+
+### BIRDSONG_BACKEND ###
+BIRDSONG_BACKEND = getattr(settings, 'BIRDSONG_BACKEND', 'birdsong.backends.smtp.SMTPEmailBackend')
+BIRDSONG_BACKEND_CLASS = module_loading.import_string(BIRDSONG_BACKEND)
+
+### BIRDSONG_SUBSCRIBE_FORM ###
+BIRDSONG_SUBSCRIBE_FORM_AJAX = getattr(settings, 'BIRDSONG_SUBSCRIBE_FORM_AJAX', True) # post form with ajax
+BIRDSONG_SUBSCRIBE_FORM_FEEDBACK = getattr(settings, 'BIRDSONG_SUBSCRIBE_FORM_FEEDBACK', True) # bootstrap5 compatible validation feedback
+BIRDSONG_SUBSCRIBE_FORM_NOVALIDATE = getattr(settings, 'BIRDSONG_SUBSCRIBE_FORM_NOVALIDATE', False) # disables built-in browser validation
+BIRDSONG_SUBSCRIBE_FORM_ID = getattr(settings, 'BIRDSONG_SUBSCRIBE_FORM_ID', 'subscribe-form') # value of the form's html id attribute
+BIRDSONG_SUBSCRIBE_FORM_BUTTON_LABEL = _(getattr(settings, 'BIRDSONG_SUBSCRIBE_FORM_BUTTON_LABEL', 'Subscribe')) # name of the form's submit button
```

Comment:
considering the entire template is configurable, do we need to go into this level of configuration? Tend to lean towards the simpler the better, there's a lot of config options added here

Reply:
The idea behind having more flexibility via options was to allow adjustments without forcing devs to overload the template. Reasoning behind that was that as a dev I would rather keep using js driven by Birdsong for forward compatibility rather than override and diverge from Birdsong's core.

But I understand the desire to keep things simple so I've removed those above mentioned options. 

Form feedback is now always on.
Built-in browser form validation is now always on (`novalidate` form attribute has been removed).
Form ID is now hardcoded to `subscrbe-form`
Form button label is now hardcoded to `Subscribe`

Furthermore, I've also removed those template override related options and instead moved templates behind `templates/birdsong` directory. This allows for a more straightforward way of override subscribe form templates (one that is in line with how wagtail and other apps handle their templates).